dotcom-rendering/src/web/lib/formatCount.test.ts

```diff
@@ -0,0 +1,16 @@
+import { formatCount } from './formatCount';
+
+describe('formatCount', () => {
+	it('formats simple numbers', () => {
+		expect(formatCount(123)).toEqual({ short: '123', long: '123' });
+	});
+	it('formats very long numbers', () => {
+		expect(formatCount(92878)).toEqual({ short: '93k', long: '92,878' });
+	});
+	it('returns zero for zero', () => {
+		expect(formatCount(0)).toEqual({ short: '0', long: '0' });
+	});
+	it('returns an elipsis for undefined', () => {
+		expect(formatCount()).toEqual({ short: '…', long: '…' });
+	});
+});
```

Comment:
I deleted `ComentCount.tests` and replaced that coverage with a more focussed set of unit tests on the actual function. This is a better use of unit tests (rather than trying to write unit tests to verify rendering logic which we can much better achieve using Storybook + Chromatic

Reply:
Maybe a little bit of pedantry here - But does count also mean any number we display? Is this a general content style rule and if so, do we group those anywhere?

Reply:
You could defo argue that `formatCount` could be a more generic function available to other parts of the code. Maybe we can think about hoisting it [into @guardian/libs](https://github.com/guardian/libs)?

Reply:
Interesting as the closest library I can see is [`timeAgo`](https://github.com/guardian/libs#timeago). Do we store content styleguide type libs anywhere? Or is @guardian/libs it?

Reply:
There's the [Guardian style guide](https://www.theguardian.com/guardian-observer-style-guide-n) which says

> numbers
> Spell out from one to nine; numerals from 10 to 999,999; thereafter use m, bn or tn for sums of money, quantities or inanimate objects in copy, eg 5m tonnes of coal, 30bn doses of vaccine, £50tn; but million or billion for people or animals, eg 1 million people, 25 million rabbits, the world population is 7 billion, etc; in headlines always use m, bn or tn

Which seems like the right reference for any wider number formatting solution.

Code wise, I think libs is it. Source has a lot of centralised styling but is not where we put utility functions like this. Although, there really does seem to be some crossover here between style and code.

@SiAdcock @sndrs what are your thoughts?

Reply:
yeah this feels like a libs to me too. i think of source at least needing to be doing _something_ will CSS, although maybe that's only in my head.

Reply:
i guess it would be something like `formatNumberAsWords`? we could group styling ones like this `formatXasX` etc